src/file_tree.ml

```diff
@@ -54,8 +54,21 @@ let load ?(extra_ignored_subtrees=Path.Set.empty) path =
     let files = String_set.of_list files in
     let ignored_sub_dirs =
       if not ignored && String_set.mem "jbuild-ignore" files then
-        String_set.of_list
-          (Io.lines_of_file (Path.to_string (Path.relative path "jbuild-ignore")))
+        let ignore_file = Path.to_string (Path.relative path "jbuild-ignore") in
+        let files =
+          Io.lines_of_file ignore_file
+        in
+        let remove_subdirs (line, valid) fn =
```

Comment:
It seems to me that this could just be a `List.filteri`? Feel free to add it to `Import.List` if it's not in the stdlib, it seems standard enough

Reply:
Done.